client/reducers/user.js

```diff
@@ -0,0 +1,37 @@
+import axios from 'axios';
+import {setError} from './index'
+
+// ACTION TYPES
+const SET_USER = 'SET_USER'
```

Comment:
what do you think about using `SET_USER` for login instead? and for sign-up specifically we'll call this `CREATE_USER` it seems like there will be two different actions here:
CREATE_USER - creates user and assigns them to the STORE user 
SET_USER - finds user and assigns them to the STORE user

Reply:
so the set user just puts the user into the store, the thunks are the ones that will be named differently to separate login and signup functions, so don't think we need to change

Reply:
Jamie and I spoke IRL - we will call the same action creator in a different thunk when logging this is 👍